mysite/mysite/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 5.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-f87&-k7lg1ie7_qzj0!zj(f!42bp2&76hc&&qwouwlad2#l(jh'
```

Comment:
As i said if you remember, things related to security should'd go to git, find how to use env variables from .env file in django and add this key to that file, then here use it.